frontend/language/src/nb.json

```
@@ -1232,7 +1232,7 @@
   "ux_editor.component_category.select": "Flervalg",
   "ux_editor.component_category.text": "Tekst",
   "ux_editor.component_deletion_confirm": "Ja, slett komponenten",
-  "ux_editor.component_deletion_text": "Er du sikker på at du vil slette denne komponenten?\nAlle Summary-komponenter knyttet til denne komponenten vil også bli slettet.",
+  "ux_editor.component_deletion_text": "Er du sikker på at du vil slette komponenten?\nAlle avhengigheter til den blir også slettet.\n",
```

```suggestion
  "ux_editor.component_deletion_text": "Er du sikker på at du vil slette komponenten?\nAlle avhengigheter til den blir også slettet.",
```
Is the `\n` at the end there for a reason?

---

no, I removed it 👍